.github/workflows/docs.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
[not blocking] evergreen would've probably been a better place for this

Reply:
[side note and blocking] let's remove the evergreen task that generates and attaches docs to the task